talk/owt/sdk/include/cpp/owt/base/localcamerastreamparameters.h

```diff
@@ -68,6 +74,7 @@ class LocalCameraStreamParameters final {
   int fps_;
   bool video_enabled_;
   bool audio_enabled_;
+  owt::ic::IntelligentCollaborationParameters ic_params_;
```

Comment:
std::optional<> this.

Reply:
Why is this necessary?

Reply:
This is by semantics, an optional param.

Reply:
`std::optional` is enabled after c++17, I am using `absl::optional`. Besides, adding optional makes the calling very strange.